.github/workflows/ci.yml

```diff
@@ -156,6 +156,12 @@ jobs:
           ]
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: "bridge/.nvmrc"
+          cache: "npm"
+          cache-dependency-path: bridge/package-lock.json
+      - run: npm --prefix bridge/ ci
```

Comment:
Can we also run the command: `npm run compile` ?

Reply:
Yeah, we can for now, but I'd like all tests to be able to run without any other steps as a goal.